component/src/main/java/org/cbioportal/genome_nexus/util/GenomicVariantUtil.java

```diff
@@ -272,13 +273,11 @@ private static String getAltFromHgvs(String hgvs, GenomicVariant.Type type) {
             case INDEL:
                 s  = getPattern("(?<=[a-z+>])[ATCG]+$", hgvs);
                 if (s == null || s.trim().length() == 0) {
-                    return "";
+                    return "-";
                 }
                 return s.trim();
             case DELETION:
-                // TODO : here, the deleted nucleotids are optional .. but we convert them into a string of "X" if they are present. Is this correct?
-                s = getPattern("(?<=[a-z+>])[ATCG]*$", hgvs);
-                return sameLengthUnknownNucleotideString(s);
+                return "-";
```

Comment:
simply returning `-` for deletions instead of same length `X`s. I think we don't need `X`s here